src/components/nodes/intermediateEvent/intermediateEvent.vue

```diff
@@ -61,6 +61,11 @@ export default {
           nodeType: 'processmaker-modeler-intermediate-message-throw-event',
           dataTest: 'switch-to-intermediate-message-throw-event',
         },
+        {
+          label: 'Intermediate Signal Catch Event',
```

Comment:
I saw this was `Signal Intermediate Catch Event` in the docs.  But it makes more sense worded this way.

Reply:
But maybe it should follow the docs anyway.

Reply:
Are you referring to the Camunda docs? https://docs.camunda.org/manual/latest/reference/bpmn20/events/signal-events/#signal-intermediate-catching-event.

I could change it to "Signal Intermediate Catching Event". I should change the rendered name as well to match.

Reply:
Ok, I found articles that reference it as:
- Signal Catching Intermediate Event
- Intermediate Catching Signal Event

It is all good.